app/api/routes/resource_retrieval.py

```diff
@@ -44,6 +44,7 @@ def get_resources():
 
     q = Resource.query
 
+
```

Comment:
I think this extra line and the other one on line 65 are causing the linter to fail. You can run `make lint` locally to check it out and then fix whatever errors it tells you about